### PR TITLE
Hot fixes for new Tour system

### PR DESCRIPTION
**Pull request by bastien-ipb**
This PR contains fixes and imps for the tour system

1. Traceback on onboarding button in user menu
2. change of button's name
3. Showing the form view before starting a tour when clicking on the list view's buttons
4. Rainbow man not showing
5. Onboarding tours were missing

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website_sale/data/website_sale_tour.xml
Comment (Feyensv) [inline addons/website_sale/data/website_sale_tour.xml:1]: `web_tours.xml`
No need to specify `website_sale` in the file name, we're already in the `website_sale` module.

File: addons/website_sale/data/website_sale_tour.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="test_01_admin_shop_tour" model="web_tour.tour">
+        <field name="name">test_01_admin_shop_tour</field>
+        <field name="sequence">130</field>
+    </record>
```
Comment (Feyensv) [inline addons/website_sale/data/website_sale_tour.xml:6]:
```suggestion
<odoo>

    <record id="test_01_admin_shop_tour" model="web_tour.tour">
        <field name="name">test_01_admin_shop_tour</field>
        <field name="sequence">130</field>
    </record>

```

Personal opinion, but it reduces conflicts and is more enjoyable to read with spaces :eyes:

**Comment by bastien-ipb** [comment]
Hello @odoo/rd-security , could I have an override ? I just rename a variable, but it was in a markup :D

**Comment by pipu-odoo** [comment]
@robodoo r+

**Comment by pipu-odoo** [comment]
@robodoo rebase-ff

**Comment by bastien-ipb** [comment]
> > Hello, could I have an override ? I just rename a variable, but it was in a markup :D
> 
> Why is this getting markup-ed at all? There seems to be only two tours using the `rainbowManMessage` feature and both `markup` it when they define it, so markup-ing the markup seems redundant.

The rainbowManMessage in JS have yet to be removed/cleaned. The rainbowManMessage is a data coming from the backend now.

**Comment by bastien-ipb** [comment]
> > The rainbowManMessage in JS have yet to be removed/cleaned. The rainbowManMessage is a data coming from the backend now.
> 
> Then it should be markup-ed at the location where it's retrieved from the backend.

Then it would be better to have the markup to three differents locations (Then the markup-ed will be saved in the localStorage), instead of the place where we are going to use the variable rainbowManMessage ?

**Comment by bastien-ipb** [comment]
> > Then it would be better to have the markup to three differents locations (Then the markup-ed will be saved in the localStorage), instead of the place where we are going to use the variable rainbowManMessage ?
> 
> Yes. That means each source can be inspected and validated, which they should be, otherwise the feature is completely worthless, you just accept any garbage without having any idea where it came from or how it came about.

Ok ok, make sense. Thx for the explanation :)

**Comment by bastien-ipb** [comment]
@xmo-odoo The change has been done, could you check pls ?

**Comment by pipu-odoo** [comment]
@robodoo rebase-ff

**Comment by pipu-odoo** [comment]
@robodoo r+

**Comment by pipu-odoo** [comment]
@robodoo r+

**Pull request closed without merging**